lib/components/DynamicFonts/index.ts

```diff
@@ -145,11 +139,31 @@ export default class DynamicFonts {
         fontsLoaded = await this.getFonts(fonts);
       } catch (error) {
         if (debug) {
-          console.log(`getFontFamily failed (try #${tryCounter}) error: ${error}`);
+          console.log(`getFontFamily failed (try #${tryCounter}) error:`, error);
```

Comment:
```suggestion
          console.log(`getFontFamily failed (try #${tryCounter}) error: `, error);
```

Reply:
It adds a space in the `log`

Reply:
Usually not

Reply:
Add `console.log('bla', 'bli');` and you'll get:
```
 LOG  bla bli
```

Reply:
Music to my ears!